### ISSUE TITLE
Tone of 5-6 articles

### ISSUE THREAD

**Issue by coke** [Raku/doc#4467]
These definitely feel like they were written by someone outside the 6 community and we pulled them in without doing a strong edit pass on them.

TODOs, "not fully docced", "this is not specifically documented".

Better to have the minimum descriptive text or comment out sections that we don't feel offer any info.

**Comment by jonathanstowe** [comment]
Yep, they were originally made by David Adler in 2015/2016 (the original material is at https://github.com/dha/perl5-to-perl6-docs,)

**Comment by coke** [comment]
bd19ab277 (simple pass)
